### PR TITLE
#63 Fixes nav mobile layout

### PR DESCRIPTION
**Pull request by batraman**
- Simplified a few classes like 'flex-sm-row' to 'flex-row'
[The -sm columns don't work below 544px hence was causing the break in the layout]
- Added the class 'flex-justify-between' to align the nav instead of using 'col-*'
[Simpler way to put the nav on either end]
- Put 'Probot' inside a span that is visible only on width >544px
[Added a custom class 'd-xs-none' that can be used with other elements as well in future]

### CODE REVIEW

**Comment by hiimbex** [comment]
Hey @batraman, thanks for this! Could you post a few screenshots of how this looks now?

**Comment by batraman** [comment]
image
This is at 320x480
image
Looks the same as before on width > 544px

File: _includes/nav.html [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 <div class="{% unless homepage %}border-bottom{% endunless %}">
   <div class="container-lg px-3">
-    <div class="py-4 d-sm-flex flex-sm-row">
-      <h1 class="alt-h4 col-sm-4 lh-default v-align-middle">
+    <div class="py-4 d-flex flex-row flex-justify-between">
```
Comment (JasonEtco) [inline _includes/nav.html]: `flex-row` is the default, so it isn't needed here.

File: assets/css/index.scss [hunk 2]
```diff
@@ -27,3 +27,9 @@ img {
 code {
   font-size: 14px;
 }
+
+@media (max-width: 543px) {
+	.d-xs-none {
```
Comment (JasonEtco) [inline assets/css/index.scss]: Could you please indent this area with space instead of tabs, for consistency?

File: assets/css/index.scss [hunk 3]
```diff
@@ -27,3 +27,9 @@ img {
...
+		display: none!important;
```
Comment (JasonEtco) [inline assets/css/index.scss]: Would you mind adding a space before `!important`? We should probably setup SCSS linting, but thats for the future 🔜

File: _includes/nav.html (hunk 1 above)
Comment (batraman) [reply _includes/nav.html]: Sorry about that. Fixed

**Comment by batraman** [comment]
Rebased from master and updated the code with style guide changes!

**Comment by JasonEtco** [comment]
Awesome, this looks great. Thank you @batraman, I'll go ahead and merge this in 🎉

**Pull request merged**